tensorflow_serving/model_servers/BUILD

```diff
@@ -186,6 +186,7 @@ pkg_tar(
     package_dir = "/usr/bin",
 )
 
+# Build with '-c opt --copt=-mavx --copt=-msse4.2'
```

Comment:
This this specified like this because of issues with march=native not pulling the correct instructions?